### PR TITLE
Fix  CSS and Add New Confirmed Tech Fair Companies

### PR DESCRIPTION
**Pull request by EarthenSky**
Address the issue from #612 where style tags were used excessively, by refactoring most of the behaviour into css classes (no more than 1 inline style statement should reasonably exist, and only in special circumstances)

Add info for our 3 confirmed companies: Arista Networks, Safe Software, and Fortinet.

Various other minor fixes as well

### CODE REVIEW

File: csss-site/src/events/tech_fair/static/tech_fair/2023/js/tech_fair_2023.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ function update(time) {
 }
 
 function updateBanner() {
-    document.getElementById("info-banner").hidden = false;
-    document.getElementById("info-banner").style.height = document.getElementById("info-text").clientHeight + 24 + 2 + "px";
+    document.getElementById("info-banner-bg").hidden = false;
+    document.getElementById("info-banner-bg").style.height = document.getElementById("info-text").clientHeight + 24 + 2 + "px";
     console.log("update banner");
```
Comment (modernNeo) [inline csss-site/src/events/tech_fair/static/tech_fair/2023/js/tech_fair_2023.js:141]: forgot to remove debug?

File: csss-site/src/events/tech_fair/static/tech_fair/2023/js/tech_fair_2023.js [hunk 2]
```diff
@@ -136,7 +136,7 @@ function update(time) {
 }
 
 function updateBanner() {
-    document.getElementById("info-banner").hidden = false;
-    document.getElementById("info-banner").style.height = document.getElementById("info-text").clientHeight + 24 + 2 + "px";
+    document.getElementById("info-banner-bg").hidden = false;
```
Comment (modernNeo) [inline csss-site/src/events/tech_fair/static/tech_fair/2023/js/tech_fair_2023.js:139]: line 67 of this file is throwing an error
Screenshot from 2023-09-16 12-18-40

**Review by modernNeo — CHANGES_REQUESTED** [review]
found 2 issues

**Comment by modernNeo** [comment]
ever going to address the necessary changes?

File: csss-site/src/events/tech_fair/static/tech_fair/2023/js/tech_fair_2023.js (hunk 1 above)
Comment (EarthenSky) [reply csss-site/src/events/tech_fair/static/tech_fair/2023/js/tech_fair_2023.js:141]: nice catch!

File: csss-site/src/events/tech_fair/static/tech_fair/2023/js/tech_fair_2023.js (hunk 2 above)
Comment (EarthenSky) [reply csss-site/src/events/tech_fair/static/tech_fair/2023/js/tech_fair_2023.js:139]: reproduced & fixed, thanks! A function was being called on the wrong page.

**Comment by EarthenSky** [comment]
added two more companies while at it (plus some small visual improvements)

**Comment by EarthenSky** [comment]
CI failed, but the pages are currently passing W3C validation:

https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-615%2Fevents%2Ftech_fair%2F2023
https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-615%2Fevents%2Ftech_fair%2F2023%2Fcompany_package

**Pull request merged**